rpcserver.go

```diff
@@ -2813,16 +2813,16 @@ func (r *rpcServer) QueryRoutes(ctx context.Context,
 func marshallRoute(route *routing.Route) *lnrpc.Route {
 	resp := &lnrpc.Route{
 		TotalTimeLock: route.TotalTimeLock,
-		TotalFees:     int64(route.TotalFees.ToSatoshis()),
-		TotalAmt:      int64(route.TotalAmount.ToSatoshis()),
+		TotalFees:     int64(route.TotalFees),
```

Comment:
The changes to `QueryRoutes` could be made its own PR, with its own set of added tests. Will make it easier to review! 😀

Reply:
@halseth Should `lnrpc.Route` be extended with additional fields `TotalFeesMSat` and `TotalAmtMSat`? Thanks!

Reply:
Hm, good question. Adding the fields wouldn't break usage of the protos, while changing the names would. So maybe just add them? :)